Calculator/Calculator/Model/Extensions/Operator+Extension.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Operator+Extension.swift
+//  Calculator
+//
+//  Created by Minseong Kang on 2023/06/09.
+//
+
+extension Operator: CalculateItem {
+	func calculate(lhs: Double, rhs: Double) -> Double {
+		var operatorsCalculateResult: Double {
+			switch self {
+			case .add:
+				return add(lhs: lhs, rhs: rhs)
+			case .subtract:
+				return subtract(lhs: lhs, rhs: rhs)
+			case .divide:
+				return divide(lhs: lhs, rhs: rhs)
+			case .multiply:
+				return multiply(lhs: lhs, rhs: rhs)
+			}
+		}
+		return operatorsCalculateResult
+	}
```

Comment:
지금의 로직대로라면, `operatorsCalculateResult`변수는 불필요해보이네요
```suggestion
    func calculate(lhs: Double, rhs: Double) -> Double {
        switch self {
        case .add:
            return add(lhs: lhs, rhs: rhs)
        case .subtract:
            return subtract(lhs: lhs, rhs: rhs)
        case .divide:
            return divide(lhs: lhs, rhs: rhs)
        case .multiply:
            return multiply(lhs: lhs, rhs: rhs)
        }
    }
	
```